### PR TITLE
Enable the twap pruning test post-upgrade.

### PR DESCRIPTION
**Pull request by phamminh0811**
Closes: #2607

## What is the purpose of the change

This PR enable the twap pruning test post-upgrade. 

## Testing and Verifying

This change is already covered by existing e2e tests.

## Documentation and Release Note

  - Does this pull request introduce a new feature or user-facing behavior changes? (yes / no)
  - Is a relevant changelog entry added to the `Unreleased` section in `CHANGELOG.md`? (yes / no)
  - How is the feature or change documented? (not applicable   /   specification (`x/<module>/spec/`)  /  [Osmosis docs repo](https://github.com/osmosis-labs/docs)   /   not documented)

### CODE REVIEW

**Comment by ValarDragon** [comment]
Are the wasm file uploads accidental? Lets delete them if so

File: tests/e2e/e2e_test.go [hunk 1]
```diff
@@ -521,16 +521,6 @@ func (s *IntegrationTestSuite) TestArithmeticTWAP() {
...
 
-	if !s.skipUpgrade {
-		// TODO: we should reduce the pruning time in the v11
-		// genesis to make this test run faster
-		// Currenty, we are testing the upgrade from v11 to v12,
-		// the pruning time is set to whatever is in the upgrade
-		// handler (two days). Therefore, we cannot reasonably
-		// test twap pruning post-upgrade.
-		s.T().Skip("skipping TWAP Pruning test. This can be re-enabled post v12")
-	}
```
Comment (ValarDragon) [inline tests/e2e/e2e_test.go:532]: Doesn't this still need to be updated in the old version genesis?

Comment (phamminh0811) [reply tests/e2e/e2e_test.go:532]: I think so. This skipUpgrade exist when upgrading from v11 to v12, but now it's no longer necessary. Should we delete them?

Comment (p0mvn) [reply tests/e2e/e2e_test.go:532]: This was needed to be updated for v11 genesis. However, this should already be present in v14:
https://github.com/osmosis-labs/osmosis/blob/v14.x/tests/e2e/initialization/config.go#L361

**Review by p0mvn — APPROVED** [review]

**Pull request merged**